explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -212,6 +210,25 @@ object ObsTabContents {
         }.getOrEmpty
       }
 
+    def constraintsSelectorFn(
+      constraintsSetId: Option[ConstraintSet.Id],
+      obsSummaryOpt:    Option[ObsSummary],
+      observations:     ConstraintsInfo
+    )(implicit
+      client:           clue.TransactionalClient[IO, explore.schemas.ObservationDB]
```

Comment:
This is great. I hadn't realized that converting a method to a function with `fn _` resolves the implicits. I was jumping through hoops all over to pass implicits to these render functions. I can simplify all that now.

Reply:
I wonder if this is still the case in Scala 3, which has context functions. I'll check that before changing anything.